js/sitnlayers.js

```diff
@@ -17,6 +17,8 @@
     var _target
     var _selectTarget
     var _drawSimpleGeom = false
+    var _map = false
+    var _markerColor
```

Comment:
Just seeing this right now... But wouldn't it be better to use `let` here ? (in order to be coherent with the usage of ES6)

Reply:
Yep yep, it's very dirty. I'll take some time to prettify this in another PR